Parsers/ASimWebSession/Parsers/ASimWebSessionBarracudaWAF.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
Can you please check below errors?
(0) Error: Missing mandatory field [TimeGenerated]
(0) Error: Missing mandatory field [Type]

Reply:
not getting these errors while doing schema test